build.gradle

```diff
@@ -2,10 +2,11 @@
 
 buildscript {
     repositories {
+        google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta6'
```

Comment:
Running in Android Studio 3 beta, it complains unless you upgrade.

Reply:
Hi @jonathan-caryl also Android Studio 2 says
`Error:This Gradle plugin requires Studio 3.0 minimum` :)
Since Android Studio 3 is beta and most of the developers still use Android Studio 2, I prefer using the latest version in Gradle 2, now it's `2.3.3`

Reply:
Fair enough. I find once you try the new Android Studio it's hard to go back to 2, as there's a lot of improvements, but I guess it is still a beta. I've switched it back to 2.3.3.